systemtests/src/main/java/io/enmasse/systemtest/executor/Exec.java

```diff
@@ -35,7 +35,7 @@
  * Class provide execution of external command
  */
 public class Exec {
-    private static Logger log = LoggerUtils.getLogger();
+    private static final Logger LOGGER = LoggerUtils.getLogger();
```

Comment:
I would prefer to keep this `log` as we have it in all the other classes as well.

Reply:
in many classes it was already renamed to a LOGGER, so if I see class with log I try ti refactor that